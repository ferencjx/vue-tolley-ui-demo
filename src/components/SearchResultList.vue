<template>
<div>
  <div id='searchResultList' v-for="jdata in results" v-bind:key=jdata.id>
    <SearchResultItem v-bind=jdata />
  </div>
</div>
</template>

<script>
import SearchResultItem from './SearchResultItem'

export default {
  name: 'SearchResultList',
  components: {
    SearchResultItem
  },
  props: {
    results: Array
  }
}

</script>
