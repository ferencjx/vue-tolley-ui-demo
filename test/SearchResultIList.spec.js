import Vue from 'vue'
import { shallowMount } from '@vue/test-utils'

import SearchResultList from '../src/components/SearchResultList'

describe('SearchResultList', () => {
  it('renders the page', () => {
    const wrapper = shallowMount(SearchResultList)
    expect(wrapper).toMatchSnapshot()
  })
  it('rendering Items', () => {
    const Constructor = Vue.extend(SearchResultList)
    const vm = new Constructor().$mount()
    console.log(vm.$el.querySelector('.SearchResultItemContent'))
    expect(vm.$el.querySelector('SearchResultItem').querySelector('.title').innerHTML).toEqual('Grant of a sub-lease')
   })
})