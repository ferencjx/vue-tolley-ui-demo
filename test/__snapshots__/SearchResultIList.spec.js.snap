// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResultList renders the page 1`] = `<div></div>`;
